Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    new_cube = cube.convolve_to(beam)
</pre><p>For large (&gt;1 GB) cubes, add a note that you may need to pass <code>cube.allow_huge_operations = True </code>for this to work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>